settings-portal/Main.vala

```diff
@@ -0,0 +1,78 @@
+/*-
+ * Copyright 2021 Alexander Mikhaylenko <alexm@gnome.org>
```

Comment:
This isn't right, considering that it's based on:

- The file chooser portal from Files
- xdg-desktop-portal-gnome - incl. my code but not a lot

So as is it's a placeholder and should be changed. Same goes for the other files.

Reply:
Also Granite for the `AccountsServiceMonitor` bits.

Reply:
I think it's correct now.